tests/functional/firefox/whatsnew/test_whatsnew_81.py

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from selenium.common.exceptions import TimeoutException
+from pages.firefox.whatsnew.whatsnew_81 import FirefoxWhatsNew81Page
+
+
+@pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
+@pytest.mark.nondestructive
+def test_send_to_device_success(base_url, selenium):
+    page = FirefoxWhatsNew81Page(selenium, base_url, params='').open()
+    assert not page.is_qr_code_displayed
+    send_to_device = page.send_to_device
+    send_to_device.type_email('success@example.com')
+    send_to_device.click_send()
+    assert send_to_device.send_successful
+
+
+@pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
+@pytest.mark.nondestructive
+def test_send_to_device_fails_when_missing_required_fields(base_url, selenium):
+    page = FirefoxWhatsNew81Page(selenium, base_url, params='').open()
+    with pytest.raises(TimeoutException):
+        page.send_to_device.click_send()
+
+
+@pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
+@pytest.mark.nondestructive
+def test_get_firefox_qr_code(base_url, selenium):
```

Comment:
This test is going to fail on stage / prod unless the translation for `sv-SE` is active.